AzureCalling/Controllers/InviteViewController.swift

```diff
@@ -114,6 +115,11 @@ class InviteViewController: UIViewController {
 
     private func onContinueButtonTapped() {
         let callConfig = JoinCallConfig(joinId: groupCallId, displayName: displayName ?? "", callType: .groupCall)
-        callingContext.startCallComposite(callConfig)
+        busyOverlay.presentIn(view: view)
+        self.callingContext.startCallComposite(callConfig, completionHandler: { [weak self] in
+            DispatchQueue.main.async {
```

Comment:
You should not need this - onContinueButtonTapped is called from the button action, which is from the main thread.